source/synthDrivers/sapi4.py

```diff
@@ -67,7 +68,7 @@ def IUnknown_Release(self, this, *args, **kwargs):
 class SynthDriver(SynthDriver):
 
 	name="sapi4"
-	description="Microsoft Speech API version 4"
+	description="NewSpeech 4"

```

Comment:
Why this change?

Reply:
I'm very sorry, this is my mistake, it has been changed back to the original description by me